### DISCUSSION TITLE
lpcXpresso55s16 USART support for testing RJ12 functionality

### DISCUSSION THREAD

**Discussion by ArjunDinesh007** [zephyrproject-rtos/zephyr#60813] [Q&A] [answered]
Hi Team,

Please help me to provide the configuration steps and sample codes of USART transmission and reception of lpcXpresso55s16 for testing RJ12 functionality

**Comment by DerekSnell** [comment, marked as answer]
Hi @ArjunDinesh007 ,

I do not know how using the RJ12 connector impacts the UART configuration.  But the Zephyr tree has several samples and tests using the UART/serial driver, including the [UART driver sample](https://docs.zephyrproject.org/latest/samples/drivers/uart/echo_bot/README.html) and these [UART driver tests](https://github.com/zephyrproject-rtos/zephyr/tree/main/tests/drivers/uart).  And this doc page details [configuring the UART driver](https://docs.zephyrproject.org/latest/hardware/peripherals/uart.html).

Best regards
